inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_binary_elementwise.cpp

```diff
@@ -2,112 +2,131 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include "dsr_tests_common.hpp"
+
 #include <functional_test_utils/layer_test_utils.hpp>
 #include <ngraph_functions/builders.hpp>
 #include <vpu/ngraph/operations/dynamic_shape_resolver.hpp>
 
 namespace {
 
-using DataType = ngraph::element::Type_t;
-using DataDims = ngraph::Shape;
+using namespace LayerTestsUtils::vpu;
+
+struct BinaryEltwiseShapes {
+    DataShapeWithUpperBound inputShape0;
+    DataShapeWithUpperBound inputShape1;
```

Comment:
Nitpick: using `lhs` and `rhs` as naming convention for such cases makes code a bit more compliance with C++ conventions..

Reply:
Done